### PR TITLE
Chapter1/homework/4

### PR DESCRIPTION
**Pull request by mihailfox**

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4881433&bid=16175205)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                Console.WriteLine($"{messageBmi}{Environment.NewLine}{messageWeight}");
+                return -1;
+            }
+            else if (heigth <= 0)
+            {
+                Console.WriteLine($"{messageBmi}{Environment.NewLine}{messageHeight}");
+                return -1;
+            }
+            else if (heigth < 0)
+            {
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Lesson3.cs]: This else case will never fire.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4881433&bid=16175205)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+	public static class Lesson3
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4881433)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (mihailfox) [reply Src/BootCamp.Chapter/Lesson3.cs]: Fixed

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4881433&bid=16175205)

**Pull request closed without merging**